package.json

```diff
@@ -2,113 +2,132 @@
+    "emoji-regex": "^10.2.1",
+    "estree-util-value-to-estree": "^3.1.1",
     "gemoji": "^8.1.0",
+    "hast-util-from-html": "^2.0.1",
     "hast-util-sanitize": "^4.0.0",
+    "hast-util-to-xast": "^3.0.0",
+    "hastscript": "^9.0.0",
     "lodash.escape": "^4.0.1",
     "lodash.kebabcase": "^4.1.1",
-    "mdast-util-toc": "^5.1.0",
-    "parse-entities": "^4.0.1",
+    "mdast-util-find-and-replace": "^3.0.1",
+    "mdast-util-phrasing": "^4.1.0",
     "path-browserify": "^1.0.1",
     "process": "^0.11.10",
     "prop-types": "^15.8.1",
-    "rehype-raw": "^5.1.0",
-    "rehype-react": "^6.2.1",
-    "rehype-sanitize": "^4.0.0",
-    "rehype-stringify": "^6.0.0",
-    "remark-breaks": "^1.0.0",
-    "remark-disable-tokenizers": "^1.0.24",
-    "remark-frontmatter": "^2.0.0",
-    "remark-parse": "^8.0.0",
-    "remark-rehype": "^8.1.0",
-    "remark-slug": "^6.0.0",
-    "remark-stringify": "^8.0.0",
-    "unified": "^9.2.2",
+    "rehype-remark": "^10.0.0",
+    "rehype-slug": "^6.0.0",
+    "remark": "^15.0.1",
+    "remark-frontmatter": "^5.0.0",
+    "remark-gfm": "^4.0.0",
+    "remark-mdx": "^3.0.0",
+    "remark-parse": "^11.0.0",
+    "trim": "^1.0.1",
+    "unified": "^11.0.4",
     "unist-util-flatmap": "^1.0.0",
-    "unist-util-map": "^3.1.2",
-    "unist-util-select": "^4.0.0",
-    "unist-util-visit": "^4.1.1"
+    "unist-util-visit": "^5.0.0",
+    "util": "^0.12.5",
+    "xast-util-to-xml": "^4.0.0"
   },
   "peerDependencies": {
-    "@readme/variable": "^16.0.0",
+    "@mdx-js/react": "^3.0.0",
+    "@readme/variable": "^16.1.0",
     "@tippyjs/react": "^4.1.0",
     "react": "16.x || 17.x || 18.x",
     "react-dom": "16.x || 17.x || 18.x"
   },
   "devDependencies": {
-    "@babel/core": "^7.24.0",
-    "@babel/eslint-parser": "^7.23.10",
+    "@babel/core": "^7.21.4",
+    "@babel/eslint-parser": "^7.21.3",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
-    "@babel/plugin-proposal-object-rest-spread": "^7.19.4",
-    "@babel/plugin-proposal-optional-chaining": "^7.18.9",
-    "@babel/plugin-transform-runtime": "^7.24.0",
-    "@babel/preset-env": "^7.24.0",
-    "@babel/preset-react": "^7.23.3",
-    "@commitlint/cli": "^19.0.3",
-    "@commitlint/config-conventional": "^19.0.3",
-    "@readme/eslint-config": "^13.5.0",
-    "@readme/stylelint-config": "^6.1.0",
+    "@babel/plugin-proposal-export-default-from": "^7.23.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
+    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
+    "@babel/plugin-proposal-private-methods": "^7.18.6",
+    "@babel/plugin-transform-runtime": "^7.21.4",
+    "@babel/preset-env": "^7.21.4",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.4",
+    "@commitlint/cli": "^17.5.1",
+    "@commitlint/config-angular": "^17.4.4",
+    "@commitlint/config-conventional": "^17.4.4",
+    "@readme/eslint-config": "^14.0.0",
+    "@readme/markdown-legacy": "npm:@readme/markdown@^6.84.0",
```

Comment:
without doing a v7 release this legacy package import is likely going to start importing the all of this new work

Reply:
Once the migration is done, I can probably delete that.